ProjectLighthouse.Servers.Website/Pages/UserSettingsPage.cshtml

```diff
@@ -140,8 +141,13 @@ function onSubmit(e){
     let selectedAvatar = "";
     document.getElementById("avatar").addEventListener("change", e => {
       const file = e.target.files.item(0);
+      const img = System.Drawing.Image.FromFile(file);
+      const minSize = 1;
+      const maxSize = 512;
       if (file.type !== "image/jpeg" && file.type !== "image/png")
           return;
+      if (!(img.Height <= maxSize && img.Height >= minSize) && (img.Width !== img.Height))
```

Comment:
This restricts images to a perfect 1:1 aspect ratio and most images aren't perfect squares so a lot of people would have to edit their images to actually use them. This could be changed to allow a 1:1 aspect ratio plus or minus some deviation, or it could be removed entirely to allow the website's CSS to properly scale the image.

Reply:
doesn't LBP actually allow non 1:1 images to be used as PFPs? i don't see why the pictures submitted have to be converted to 1:1 if the game already handles everything